### PR TITLE
Configurable BigLearn API URLs

### PR DESCRIPTION
**Pull request by joemsak**
https://www.pivotaltracker.com/story/show/91198902

### CODE REVIEW

File: app/external/openstax/big_learn/v1/real_client.rb [hunk 1]
```diff
@@ -1,30 +1,18 @@
 class OpenStax::BigLearn::V1::RealClient
-
-  include Singleton
+  def initialize(configuration)
+    @client_id = configuration.client_id
+    @secret = configuration.secret
+    @server_url = configuration.server_url
+    @oauth_client = OAuth2::Client.new(@client_id, @secret, site: @server_url)
```
Comment (jpslav) [inline app/external/openstax/big_learn/v1/real_client.rb]: BigLearn isn't reached via oauth, so these lines can go.

Comment (joemsak) [inline app/external/openstax/big_learn/v1/real_client.rb]: Sweet will update with another PR as this has been merged. I strictly copied Dante's work as noted in the PT : )

**Comment by jpslav** [comment]
Yep, all good, just can remove since unused for now.  Thanks @Dantemss for the earlier merge.

File: app/external/openstax/big_learn/v1/real_client.rb (hunk 1 above)
Comment (jpslav) [inline app/external/openstax/big_learn/v1/real_client.rb]: Yep, all good, just can remove since unused for now.

**Comment by jpslav** [comment]
Thanks @Dantemss for the earlier merge.

**Pull request merged**